## 48% of end-end tests have passed
```
Total executed tests: 56
Failed tests: 27
Expected to fail tests: 2
Passed tests: 27
```
```
IDs of failed tests: 727230,702809,702782,702733,703194,702859,702851,702840,703086,702894,702745,702850,704615,703496,702855,703202,703133,702786,702839,702838,740221,702844,702843,702731,702730,702958,702841 
```
```
IDs of expected to fail tests: 702948,703629 
```
<h3>Failed tests (27)</h3><details><summary>Click to expand</summary><br/><li><a href="https://ci.status.im/job/status-mobile/job/e2e/job/status-app-prs-rerun/parambuild/?BRANCH_NAME=fix/no_log2&PR_ID=21806&APK_NAME=21806.apk&TR_CASE_IDS=727230,702809,702782,702733,703194,702859,702851,702840,703086,702894,702745,702850,704615,703496,702855,703202,703133,702786,702839,702838,740221,702844,702843,702731,702730,702958,702841">Rerun failed tests</a></li><br/><h4>Class TestFallbackMultipleDevice:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3749330">test_fallback_with_correct_seed_phrase</a>, id: 740221 </b></td></tr><tr><td><p><blockquote><div>Device 1: Looking for a message by text: Test message </div><div>Device 2: Looking for a message by text: Test message </div></blockquote></p>

```
critical/test_fallback.py:161: in test_fallback_with_correct_seed_phrase
    self.errors.verify_no_errors()
base_test_case.py:192: in verify_no_errors
    pytest.fail('\n '.join([self.errors.pop(0) for _ in range(len(self.errors))]))
 Pair button is absent for the device 2 inside Paired devices list of profile 1 before pairing
```

<br/><br/></td></tr></tbody></table><h4>Class TestGroupChatMultipleDeviceMergedNewUI:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3749315">test_group_chat_reactions</a>, id: 703202 </b></td></tr><tr><td><p><blockquote><div>Device 1: Find `EmojisNumber` by `xpath`: `//*[starts-with(@text,'This is a test message to check some reactions.')]/ancestor::android.view.ViewGroup[@content-desc='chat-item']/../..//*[@content-desc='emoji-reaction-2']/android.widget.TextView[2]` </div><div>Device 1: Find `EmojisNumber` by `xpath`: `//*[starts-with(@text,'This is a test message to check some reactions.')]/ancestor::android.view.ViewGroup[@content-desc='chat-item']/../..//*[@content-desc='emoji-reaction-2']/android.widget.TextView[2]` </div></blockquote></p>

```
critical/chats/test_group_chat.py:131: in test_group_chat_reactions
    self.loop.run_until_complete(run_in_parallel((
/usr/lib/python3.10/asyncio/base_events.py:649: in run_until_complete
    return future.result()
__init__.py:52: in run_in_parallel
    returns.append(await k)
/usr/lib/python3.10/concurrent/futures/thread.py:58: in run
    result = self.fn(*self.args, **self.kwargs)
critical/chats/test_group_chat.py:127: in _check_reactions_count
    chat_element.emojis_below_message(emoji="thumbs-up").wait_for_element_text(2)
../views/base_element.py:190: in wait_for_element_text
    self.driver.fail(message if message else "`%s` is not equal to expected `%s` in %s sec" % (
base_test_case.py:180: in fail
    pytest.fail('Device %s: %s' % (self.number, text))
 Device 1: `1` is not equal to expected `2` in 30 sec
```

<br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/19ef8f2b0a3e4705b9e25fa30a1005b4?auth=f72fdd9ace474448b32e5398f62108fc#188">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/04ef7f820f79423690880f4ee633348c?auth=145e05055aeed9e4e28d8b31d9c21d63#118">Steps, video, logs</a></li></ul></p><p>Device 3:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/cba08f4004dc4692a7c95941876e07cb?auth=d6175b9d2817c717890fcb34f08c7444#103">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestActivityCenterContactRequestMultipleDevicePR:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3749303">test_activity_center_contact_request_accept_swipe_mark_all_as_read</a>, id: 702851 </b></td></tr><tr><td><p><blockquote><div>Device 2: Find `Button` by `accessibility id`: `tab-contacts` </div><div>Device 2: Tap on found: Button </div></blockquote></p>

```
activity_center/test_activity_center.py:147: in test_activity_center_contact_request_accept_swipe_mark_all_as_read
    self.errors.verify_no_errors()
base_test_case.py:192: in verify_no_errors
    pytest.fail('\n '.join([self.errors.pop(0) for _ in range(len(self.errors))]))
 Contact was not added to contact list after accepting contact request (as sender)
```

<br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/11f23bb88e124433b7b877fc40798a0a?auth=19fb23696ecde0d8d0735ecc8eb6b992#85">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/fa2fced2668b4c21940dc98f57f3995a?auth=3a9b9e76690e05b654ab261d073aca00#96">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3749302">test_activity_center_contact_request_decline</a>, id: 702850 </b></td></tr><tr><td><p><blockquote><div>Device 1: Find `Button` by `accessibility id`: `tab-contacts` </div><div>Device 1: Find `Text` by `xpath`: `//*[@content-desc="pending-contact-requests-count"]/android.widget.TextView` </div></blockquote></p>

```
activity_center/test_activity_center.py:72: in test_activity_center_contact_request_decline
    if self.home_1.pending_contact_request_text.text != '1':
../views/base_element.py:432: in text
    text = self.find_element().text
../views/base_element.py:79: in find_element
    raise NoSuchElementException(
 Device 1: Text by xpath: `//*[@content-desc="pending-contact-requests-count"]/android.widget.TextView` is not found on the screen; For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
```

<br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/11f23bb88e124433b7b877fc40798a0a?auth=19fb23696ecde0d8d0735ecc8eb6b992#66">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/fa2fced2668b4c21940dc98f57f3995a?auth=3a9b9e76690e05b654ab261d073aca00#50">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestWalletMultipleDevice:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3749326">test_wallet_send_asset_from_drawer</a>, id: 727230 </b></td></tr><tr><td><p><blockquote><div>Device 2: Find `Text` by `xpath`: `//android.view.ViewGroup[@content-desc='container']/android.widget.TextView[@text='Ether']/../android.widget.TextView[3]` </div><div>Device 2: `Text` is `0.14259 ETH` </div></blockquote></p>

```
critical/test_wallet.py:187: in test_wallet_send_asset_from_drawer
    self.errors.verify_no_errors()
base_test_case.py:192: in verify_no_errors
    pytest.fail('\n '.join([self.errors.pop(0) for _ in range(len(self.errors))]))
 Sender balance is not updated on Etherscan, it is 0.3368 but expected to be 0.3369
```

<br/><br/></td></tr></tbody></table><h4>Class TestCommunityMultipleDeviceMerged:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3749314">test_community_several_images_send_reply</a>, id: 703194 </b></td></tr><tr><td>

```
Test setup failed: critical/chats/test_public_chat_browsing.py:345: in prepare_devices
    self.community_2.join_community()
../views/chat_view.py:419: in join_community
    ChatView(self.driver).chat_element_by_text("https://status.app/c/").click_on_link_inside_message_body()
../views/chat_view.py:175: in click_on_link_inside_message_body
    self.message_body.wait_for_visibility_of_element(30)
../views/base_element.py:147: in wait_for_visibility_of_element
    raise TimeoutException(
 Device 2: Text by xpath:`//android.view.ViewGroup[@content-desc='chat-item']//android.widget.TextView[contains(@text,'https://status.app/c/')]` is not found on the screen after wait_for_visibility_of_element
```

<br/><br/></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3749305">test_community_one_image_send_reply</a>, id: 702859 </b></td></tr><tr><td>

```
Test setup failed: critical/chats/test_public_chat_browsing.py:345: in prepare_devices
    self.community_2.join_community()
../views/chat_view.py:419: in join_community
    ChatView(self.driver).chat_element_by_text("https://status.app/c/").click_on_link_inside_message_body()
../views/chat_view.py:175: in click_on_link_inside_message_body
    self.message_body.wait_for_visibility_of_element(30)
../views/base_element.py:147: in wait_for_visibility_of_element
    raise TimeoutException(
 Device 2: Text by xpath:`//android.view.ViewGroup[@content-desc='chat-item']//android.widget.TextView[contains(@text,'https://status.app/c/')]` is not found on the screen after wait_for_visibility_of_element
```

<br/><br/></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3749296">test_community_emoji_send_copy_paste_reply</a>, id: 702840 </b></td></tr><tr><td>

```
Test setup failed: critical/chats/test_public_chat_browsing.py:345: in prepare_devices
    self.community_2.join_community()
../views/chat_view.py:419: in join_community
    ChatView(self.driver).chat_element_by_text("https://status.app/c/").click_on_link_inside_message_body()
../views/chat_view.py:175: in click_on_link_inside_message_body
    self.message_body.wait_for_visibility_of_element(30)
../views/base_element.py:147: in wait_for_visibility_of_element
    raise TimeoutException(
 Device 2: Text by xpath:`//android.view.ViewGroup[@content-desc='chat-item']//android.widget.TextView[contains(@text,'https://status.app/c/')]` is not found on the screen after wait_for_visibility_of_element
```

<br/><br/></td></tr><tr><td><b>4. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3749312">test_community_mark_all_messages_as_read</a>, id: 703086 </b></td></tr><tr><td>

```
Test setup failed: critical/chats/test_public_chat_browsing.py:345: in prepare_devices
    self.community_2.join_community()
../views/chat_view.py:419: in join_community
    ChatView(self.driver).chat_element_by_text("https://status.app/c/").click_on_link_inside_message_body()
../views/chat_view.py:175: in click_on_link_inside_message_body
    self.message_body.wait_for_visibility_of_element(30)
../views/base_element.py:147: in wait_for_visibility_of_element
    raise TimeoutException(
 Device 2: Text by xpath:`//android.view.ViewGroup[@content-desc='chat-item']//android.widget.TextView[contains(@text,'https://status.app/c/')]` is not found on the screen after wait_for_visibility_of_element
```

<br/><br/></td></tr><tr><td><b>5. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3749307">test_community_contact_block_unblock_offline</a>, id: 702894 </b></td></tr><tr><td>

```
Test setup failed: critical/chats/test_public_chat_browsing.py:345: in prepare_devices
    self.community_2.join_community()
../views/chat_view.py:419: in join_community
    ChatView(self.driver).chat_element_by_text("https://status.app/c/").click_on_link_inside_message_body()
../views/chat_view.py:175: in click_on_link_inside_message_body
    self.message_body.wait_for_visibility_of_element(30)
../views/base_element.py:147: in wait_for_visibility_of_element
    raise TimeoutException(
 Device 2: Text by xpath:`//android.view.ViewGroup[@content-desc='chat-item']//android.widget.TextView[contains(@text,'https://status.app/c/')]` is not found on the screen after wait_for_visibility_of_element
```

<br/><br/></td></tr><tr><td><b>6. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3749324">test_community_edit_delete_message_when_offline</a>, id: 704615 </b></td></tr><tr><td>

```
Test setup failed: critical/chats/test_public_chat_browsing.py:345: in prepare_devices
    self.community_2.join_community()
../views/chat_view.py:419: in join_community
    ChatView(self.driver).chat_element_by_text("https://status.app/c/").click_on_link_inside_message_body()
../views/chat_view.py:175: in click_on_link_inside_message_body
    self.message_body.wait_for_visibility_of_element(30)
../views/base_element.py:147: in wait_for_visibility_of_element
    raise TimeoutException(
 Device 2: Text by xpath:`//android.view.ViewGroup[@content-desc='chat-item']//android.widget.TextView[contains(@text,'https://status.app/c/')]` is not found on the screen after wait_for_visibility_of_element
```

<br/><br/></td></tr><tr><td><b>7. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3749295">test_community_message_delete</a>, id: 702839 </b></td></tr><tr><td>

```
Test setup failed: critical/chats/test_public_chat_browsing.py:345: in prepare_devices
    self.community_2.join_community()
../views/chat_view.py:419: in join_community
    ChatView(self.driver).chat_element_by_text("https://status.app/c/").click_on_link_inside_message_body()
../views/chat_view.py:175: in click_on_link_inside_message_body
    self.message_body.wait_for_visibility_of_element(30)
../views/base_element.py:147: in wait_for_visibility_of_element
    raise TimeoutException(
 Device 2: Text by xpath:`//android.view.ViewGroup[@content-desc='chat-item']//android.widget.TextView[contains(@text,'https://status.app/c/')]` is not found on the screen after wait_for_visibility_of_element
```

<br/><br/></td></tr><tr><td><b>8. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3749294">test_community_message_send_check_timestamps_sender_username</a>, id: 702838 </b></td></tr><tr><td><p><blockquote><div>Device 2: Joining community </div><div>Device 2: Looking for a message by text: https://status.app/c/ </div></blockquote></p>

```
Test setup failed: critical/chats/test_public_chat_browsing.py:345: in prepare_devices
    self.community_2.join_community()
../views/chat_view.py:419: in join_community
    ChatView(self.driver).chat_element_by_text("https://status.app/c/").click_on_link_inside_message_body()
../views/chat_view.py:175: in click_on_link_inside_message_body
    self.message_body.wait_for_visibility_of_element(30)
../views/base_element.py:147: in wait_for_visibility_of_element
    raise TimeoutException(
 Device 2: Text by xpath:`//android.view.ViewGroup[@content-desc='chat-item']//android.widget.TextView[contains(@text,'https://status.app/c/')]` is not found on the screen after wait_for_visibility_of_element
```

<br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/6a418044ce524b9baf144c8b9e709a00?auth=2b7f815e38265b790645b1e1ef064170">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/8fdbd514545d4c258a1095266e796e2e?auth=9dd9eefdc0ca83fcd5a6c89e85d6669f">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>9. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3749299">test_community_links_with_previews_github_youtube_twitter_gif_send_enable</a>, id: 702844 </b></td></tr><tr><td>

```
Test setup failed: critical/chats/test_public_chat_browsing.py:345: in prepare_devices
    self.community_2.join_community()
../views/chat_view.py:419: in join_community
    ChatView(self.driver).chat_element_by_text("https://status.app/c/").click_on_link_inside_message_body()
../views/chat_view.py:175: in click_on_link_inside_message_body
    self.message_body.wait_for_visibility_of_element(30)
../views/base_element.py:147: in wait_for_visibility_of_element
    raise TimeoutException(
 Device 2: Text by xpath:`//android.view.ViewGroup[@content-desc='chat-item']//android.widget.TextView[contains(@text,'https://status.app/c/')]` is not found on the screen after wait_for_visibility_of_element
```

<br/><br/></td></tr><tr><td><b>10. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3749298">test_community_message_edit</a>, id: 702843 </b></td></tr><tr><td>

```
Test setup failed: critical/chats/test_public_chat_browsing.py:345: in prepare_devices
    self.community_2.join_community()
../views/chat_view.py:419: in join_community
    ChatView(self.driver).chat_element_by_text("https://status.app/c/").click_on_link_inside_message_body()
../views/chat_view.py:175: in click_on_link_inside_message_body
    self.message_body.wait_for_visibility_of_element(30)
../views/base_element.py:147: in wait_for_visibility_of_element
    raise TimeoutException(
 Device 2: Text by xpath:`//android.view.ViewGroup[@content-desc='chat-item']//android.widget.TextView[contains(@text,'https://status.app/c/')]` is not found on the screen after wait_for_visibility_of_element
```

<br/><br/></td></tr><tr><td><b>11. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3749297">test_community_unread_messages_badge</a>, id: 702841 </b></td></tr><tr><td>

```
Test setup failed: critical/chats/test_public_chat_browsing.py:345: in prepare_devices
    self.community_2.join_community()
../views/chat_view.py:419: in join_community
    ChatView(self.driver).chat_element_by_text("https://status.app/c/").click_on_link_inside_message_body()
../views/chat_view.py:175: in click_on_link_inside_message_body
    self.message_body.wait_for_visibility_of_element(30)
../views/base_element.py:147: in wait_for_visibility_of_element
    raise TimeoutException(
 Device 2: Text by xpath:`//android.view.ViewGroup[@content-desc='chat-item']//android.widget.TextView[contains(@text,'https://status.app/c/')]` is not found on the screen after wait_for_visibility_of_element
```

<br/><br/></td></tr></tbody></table><h4>Class TestOneToOneChatMultipleSharedDevicesNewUiTwo:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3749320">test_1_1_chat_mute_chat</a>, id: 703496 </b></td></tr><tr><td><p><blockquote><div>Device 1: `PreveiewMessageText` is `test message ` </div><div>Device 1: Click until `ChatMessageInput` by `accessibility id`: `chat-message-input` will be presented </div></blockquote></p>

```
critical/chats/test_1_1_public_chats.py:639: in test_1_1_chat_mute_chat
    chat.click()
../views/home_view.py:64: in click
    self.click_until_presence_of_element(desired_element=desired_element)
../views/base_element.py:111: in click_until_presence_of_element
    el.click()
../../../../status-app-prs-rerun@tmp/venv/lib/python3.10/site-packages/selenium/webdriver/remote/webelement.py:94: in click
    self._execute(Command.CLICK_ELEMENT)
../../../../status-app-prs-rerun@tmp/venv/lib/python3.10/site-packages/selenium/webdriver/remote/webelement.py:395: in _execute
    return self._parent.execute(command, params)
../../../../status-app-prs-rerun@tmp/venv/lib/python3.10/site-packages/selenium/webdriver/remote/webdriver.py:345: in execute
    self.error_handler.check_response(response)
../../../../status-app-prs-rerun@tmp/venv/lib/python3.10/site-packages/appium/webdriver/errorhandler.py:122: in check_response
    raise exception_class(msg=message, stacktrace=format_stacktrace(stacktrace))
 The element 'By.xpath: //*[@content-desc='author-primary-name'][starts-with(@text,'shtPrg5TZpYJ427s1iQR')]/..' is not linked to the same object in DOM anymore; For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#stale-element-reference-exception
E   Stacktrace:
E   io.appium.uiautomator2.common.exceptions.StaleElementReferenceException: The element 'By.xpath: //*[@content-desc='author-primary-name'][starts-with(@text,'shtPrg5TZpYJ427s1iQR')]/..' is not linked to the same object in DOM anymore
E   	at io.appium.uiautomator2.model.ElementsCache.restore(ElementsCache.java:122)
E   	at io.appium.uiautomator2.model.ElementsCache.get(ElementsCache.java:153)
E   	at io.appium.uiautomator2.handler.Click.safeHandle(Click.java:36)
E   	at io.appium.uiautomator2.handler.request.SafeRequestHandler.handle(SafeRequestHandler.java:59)
E   	at io.appium.uiautomator2.server.AppiumServlet.handleRequest(AppiumServlet.java:259)
E   	at io.appium.uiautomator2.server.AppiumServlet.handleHttpRequest(AppiumServlet.java:253)
E   	at io.appium.uiautomator2.http.ServerHandler.channelRead(ServerHandler.java:77)
E   	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:366)
E   	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:352)
E   	at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:345)
E   	at io.netty.handler.codec.MessageToMessageDecoder.channelRead(MessageToMessageDecoder.java:102)
E   	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:366)
E   	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:352)
E   	at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:345)
E   	at io.netty.channel.CombinedChannelDuplexHandler$DelegatingChannelHandlerContext.fireChannelRead(CombinedChannelDuplexHandler.java:435)
E   	at io.netty.handler.codec.ByteToMessageDecoder.fireChannelRead(ByteToMessageDecoder.java:293)
E   	at io.netty.handler.codec.ByteToMessageDecoder.channelRead(ByteToMessageDecoder.java:267)
E   	at io.netty.channel.CombinedChannelDuplexHandler.channelRead(CombinedChannelDuplexHandler.java:250)
E   	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:366)
E   	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:352)
E   	at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:345)
E   	at io.netty.handler.timeout.IdleStateHandler.channelRead(IdleStateHandler.java:266)
E   	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:366)
E   	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:352)
E   	at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:345)
E   	at io.netty.channel.DefaultChannelPipeline$HeadContext.channelRead(DefaultChannelPipeline.java:1294)
E   	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:366)
E   	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:352)
E   	at io.netty.channel.DefaultChannelPipeline.fireChannelRead(DefaultChannelPipeline.java:911)
E   	at io.netty.channel.nio.AbstractNioByteChannel$NioByteUnsafe.read(AbstractNioByteChannel.java:131)
E   	at io.netty.channel.nio.NioEventLoop.processSelectedKey(NioEventLoop.java:611)
E   	at io.netty.channel.nio.NioEventLoop.processSelectedKeysOptimized(NioEventLoop.java:552)
E   	at io.netty.channel.nio.NioEventLoop.processSelectedKeys(NioEventLoop.java:466)
E   	at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:438)
E   	at io.netty.util.concurrent.SingleThreadEventExecutor$2.run(SingleThreadEventExecutor.java:140)
E   	at io.netty.util.concurrent.DefaultThreadFactory$DefaultRunnableDecorator.run(DefaultThreadFactory.java:144)
E   	at java.lang.Thread.run(Thread.java:1012)
```

<br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/3b3ec6cf1ff94d7a8e1c7e8106d2b765?auth=b117376a1c76afee3c638413797e88e1#114">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/862b288c21e14fde93858cb9a8c1710a?auth=8b69279b4f21031defba401367a75499#124">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestCommunityOneDeviceMerged:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3749313">test_restore_multiaccount_with_waku_backup_remove_profile_switch</a>, id: 703133 </b></td></tr><tr><td><p><blockquote><div>Device 1: Tap on found: Button </div><div># STEP: Check that removed user is not shown in the list anymore</div></blockquote></p>

```
critical/chats/test_public_chat_browsing.py:240: in test_restore_multiaccount_with_waku_backup_remove_profile_switch
    self.errors.verify_no_errors()
base_test_case.py:192: in verify_no_errors
    pytest.fail('\n '.join([self.errors.pop(0) for _ in range(len(self.errors))]))
 zQ3...dWXh5 was not restored as a contact from waku backup!
E    zQ3...Vacac was not restored as a contact from waku backup!
E    admin_open was not restored from waku-backup!!
E    member_open was not restored from waku-backup!!
E    admin_closed was not restored from waku-backup!!
E    member_closed was not restored from waku-backup!!
```

<br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/769d036bfade40e5b1b382f228663350?auth=8fd275a541599d0f0342d555a3f20fed#243">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestActivityMultipleDevicePRTwo:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3749311">test_activity_center_admin_notification_accept_swipe</a>, id: 702958 </b></td></tr><tr><td><p><blockquote><div>Device 2: Joining community </div><div>Device 2: Looking for a message by text: https://status.app/c/ </div></blockquote></p>

```
activity_center/test_activity_center.py:462: in test_activity_center_admin_notification_accept_swipe
    self.community_2.join_community(open_community=False)
../views/chat_view.py:419: in join_community
    ChatView(self.driver).chat_element_by_text("https://status.app/c/").click_on_link_inside_message_body()
../views/chat_view.py:175: in click_on_link_inside_message_body
    self.message_body.wait_for_visibility_of_element(30)
../views/base_element.py:147: in wait_for_visibility_of_element
    raise TimeoutException(
 Device 2: Text by xpath:`//android.view.ViewGroup[@content-desc='chat-item']//android.widget.TextView[contains(@text,'https://status.app/c/')]` is not found on the screen after wait_for_visibility_of_element
```

<br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/0e8cc71c9c8942b483318b2474a1518b?auth=bb61d70b0641846bc9c1752f661fcd18#164">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/e8780f77cb4a44a9ad8cd2c3f53b6f51?auth=4f80529903cf9280a9731402ea8ed3f6#115">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestOneToOneChatMultipleSharedDevicesNewUi:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3749286">test_1_1_chat_emoji_send_reply_and_open_link</a>, id: 702782 </b></td></tr><tr><td><p><blockquote><div>Device 2: Find `OpenInStatusButton` by `xpath`: `//*[@text="Open in Status"]` </div><div>Device 2: Tap on found: OpenInStatusButton </div></blockquote></p>

```
critical/chats/test_1_1_public_chats.py:178: in test_1_1_chat_emoji_send_reply_and_open_link
    self.errors.verify_no_errors()
base_test_case.py:192: in verify_no_errors
    pytest.fail('\n '.join([self.errors.pop(0) for _ in range(len(self.errors))]))
 Message with emoji was not sent or received in 1-1 chat
E    No reply received in 1-1 chat
E    Reply for 'Test with link: https://status.im/ here should be nothing unusual.' not present in message received in public chat
E    Link message reaction is not shown for the sender
```

<br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/e491fe32e7204d4e8b6c53d522986406?auth=e567520243dd6ecc15ac7c5fd493c2d8#125">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/51d94cb742df4405854d41116ea44dca?auth=c930d15411f24e910f99c78a97d264ee#107">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3749281">test_1_1_chat_text_message_delete_push_disappear</a>, id: 702733 </b></td></tr><tr><td><p><blockquote><div>Device 2: Find `Text` by `xpath`: `//*[starts-with(@text,'smth I should edit')]/ancestor::android.view.ViewGroup[@content-desc='chat-item']//*[@content-desc='message-status']/android.widget.TextView` </div><div>Device 2: `Text` is `Sent` </div></blockquote></p>

```
critical/chats/test_1_1_public_chats.py:485: in test_1_1_chat_text_message_delete_push_disappear
    self.chat_2.chat_element_by_text(message_after_edit_1_1).wait_for_status_to_be("Delivered")
../views/chat_view.py:225: in wait_for_status_to_be
    raise TimeoutException("Message status was not changed to %s, it's %s" % (expected_status, current_status))
 Message status was not changed to Delivered, it's Sent
```

<br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/e491fe32e7204d4e8b6c53d522986406?auth=e567520243dd6ecc15ac7c5fd493c2d8#449">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/51d94cb742df4405854d41116ea44dca?auth=c930d15411f24e910f99c78a97d264ee#406">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3749283">test_1_1_chat_non_latin_messages_stack_update_profile_photo</a>, id: 702745 </b></td></tr><tr><td><p><blockquote><div>Device 1: Tap on found: SendMessageButton </div><div>Device 2: Looking for a message by text: profile_photo </div></blockquote></p>

```
critical/chats/test_1_1_public_chats.py:312: in test_1_1_chat_non_latin_messages_stack_update_profile_photo
    self.chat_2.chat_element_by_text(message).wait_for_visibility_of_element(30)
../views/base_element.py:147: in wait_for_visibility_of_element
    raise TimeoutException(
 Device 2: ChatElementByText by xpath:`//*[starts-with(@text,'profile_photo')]/ancestor::android.view.ViewGroup[@content-desc='chat-item']` is not found on the screen after wait_for_visibility_of_element
```

<br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/e491fe32e7204d4e8b6c53d522986406?auth=e567520243dd6ecc15ac7c5fd493c2d8#234">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/51d94cb742df4405854d41116ea44dca?auth=c930d15411f24e910f99c78a97d264ee#196">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>4. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3749304">test_1_1_chat_edit_message</a>, id: 702855 </b></td></tr><tr><td><p><blockquote><div>Device 2: Find `Text` by `xpath`: `//*[starts-with(@text,'Message before edit 1-1')]/ancestor::android.view.ViewGroup[@content-desc='chat-item']//*[@content-desc='message-status']/android.widget.TextView` </div><div>Device 2: `Text` is `Sent` </div></blockquote></p>

```
critical/chats/test_1_1_public_chats.py:388: in test_1_1_chat_edit_message
    self.chat_2.chat_element_by_text(message_before_edit_1_1).wait_for_status_to_be("Delivered")
../views/chat_view.py:225: in wait_for_status_to_be
    raise TimeoutException("Message status was not changed to %s, it's %s" % (expected_status, current_status))
 Message status was not changed to Delivered, it's Sent
```

<br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/e491fe32e7204d4e8b6c53d522986406?auth=e567520243dd6ecc15ac7c5fd493c2d8#367">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/51d94cb742df4405854d41116ea44dca?auth=c930d15411f24e910f99c78a97d264ee#279">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>5. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3749279">test_1_1_chat_pin_messages</a>, id: 702731 </b></td></tr><tr><td><p><blockquote><div>Device 1: Find `Text` by `xpath`: `//*[starts-with(@text,'Message 1')]/ancestor::android.view.ViewGroup[@content-desc='chat-item']//*[@content-desc='message-status']/android.widget.TextView` </div><div>Device 1: `Text` is `Sent` </div></blockquote></p>

```
critical/chats/test_1_1_public_chats.py:188: in test_1_1_chat_pin_messages
    self.chat_1.chat_element_by_text(self.message_1).wait_for_status_to_be("Delivered")
../views/chat_view.py:225: in wait_for_status_to_be
    raise TimeoutException("Message status was not changed to %s, it's %s" % (expected_status, current_status))
 Message status was not changed to Delivered, it's Sent
```

<br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/e491fe32e7204d4e8b6c53d522986406?auth=e567520243dd6ecc15ac7c5fd493c2d8#183">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/51d94cb742df4405854d41116ea44dca?auth=c930d15411f24e910f99c78a97d264ee#180">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>6. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3749278">test_1_1_chat_message_reaction</a>, id: 702730 </b></td></tr><tr><td><p><blockquote><div>Device 2: Wait for text element `EmojisNumber` to be equal to `1` </div><div>Device 2: Find `EmojisNumber` by `xpath`: `//*[starts-with(@text,'Message sender')]/ancestor::android.view.ViewGroup[@content-desc='chat-item']/../..//*[@content-desc='emoji-reaction-2']/android.widget.TextView[2]` </div></blockquote></p>

```
critical/chats/test_1_1_public_chats.py:62: in test_1_1_chat_message_reaction
    message_receiver.emojis_below_message(emoji="thumbs-up").wait_for_element_text(1, 90)
../views/base_element.py:181: in wait_for_element_text
    element_text = self.find_element().text.strip()
../views/base_element.py:79: in find_element
    raise NoSuchElementException(
 Device 2: EmojisNumber by xpath: `//*[starts-with(@text,'Message sender')]/ancestor::android.view.ViewGroup[@content-desc='chat-item']/../..//*[@content-desc='emoji-reaction-2']/android.widget.TextView[2]` is not found on the screen; For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
```

<br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/e491fe32e7204d4e8b6c53d522986406?auth=e567520243dd6ecc15ac7c5fd493c2d8#103">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/51d94cb742df4405854d41116ea44dca?auth=c930d15411f24e910f99c78a97d264ee#101">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestCommunityMultipleDeviceMergedTwo:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3749292">test_community_markdown_support</a>, id: 702809 </b></td></tr><tr><td><p><blockquote><div>Device 1: Looking for a message by text: quote reply (one row) </div><div>Device 2: Looking for a message by text: quote reply (one row) </div></blockquote></p>

```
critical/chats/test_public_chat_browsing.py:1009: in test_community_markdown_support
    self.errors.verify_no_errors()
base_test_case.py:192: in verify_no_errors
    pytest.fail('\n '.join([self.errors.pop(0) for _ in range(len(self.errors))]))
 italic text in underscore is not displayed with markdown in community channel for the recipient (device 1) 
E   
E    inline code is not displayed with markdown in community channel for the recipient (device 1) 
E   
E    code blocks is not displayed with markdown in community channel for the recipient (device 1) 
E   
E    bold text in underscores is not displayed with markdown in 1-1 chat for the recipient (device 2) 
E   
E    italic text in asteric is not displayed with markdown in 1-1 chat for the recipient (device 2) 
E   
E    italic text in underscore is not displayed with markdown in 1-1 chat for the recipient (device 2) 
E   
E    code blocks is not displayed with markdown in 1-1 chat for the recipient (device 2) 
E   
E    quote reply (one row) is not displayed with markdown in 1-1 chat for the recipient (device 2)
```

<br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/7f398016b29749b68d18fea6575c3fc3?auth=027c012c7e830c4946eb916a98a1f171#201">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/662c4df8b87e4388adb93a5bd9117ece?auth=be8d26dc7a2852d597406f6c012843e6#175">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3749289">test_community_mentions_push_notification</a>, id: 702786 </b></td></tr><tr><td><p><blockquote><div>Device 2: Getting PN by 'shYYZjweYeRCxueXzjDf' </div><div>Device 2: Looking for a message by text: shYYZjweYeRCxueXzjDf </div></blockquote></p>

```
critical/chats/test_public_chat_browsing.py:957: in test_community_mentions_push_notification
    self.errors.verify_no_errors()
base_test_case.py:192: in verify_no_errors
    pytest.fail('\n '.join([self.errors.pop(0) for _ in range(len(self.errors))]))
 Push notification with the mention was not received by admin
E    Can not edit a message with a mention
```

<br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/7f398016b29749b68d18fea6575c3fc3?auth=027c012c7e830c4946eb916a98a1f171#144">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/662c4df8b87e4388adb93a5bd9117ece?auth=be8d26dc7a2852d597406f6c012843e6#129">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table></details><h3>Expected to fail tests (2)</h3><details><summary>Click to expand</summary><br/><br/><h4>Class TestCommunityMultipleDeviceMergedTwo:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3749309">test_community_hashtag_links_to_community_channels</a>, id: 702948 </b></td></tr><tr><td><p><blockquote><div>Device 2: Tap on found: SendMessageButton </div><div>Device 2: Looking for a message by text: https://status.app/c/ </div></blockquote></p>

```
critical/chats/test_public_chat_browsing.py:1045: in test_community_hashtag_links_to_community_channels
    self.chat_2.chat_element_by_text("https://status.app/c/").click_on_link_inside_message_body()
../views/chat_view.py:175: in click_on_link_inside_message_body
    self.message_body.wait_for_visibility_of_element(30)
../views/base_element.py:147: in wait_for_visibility_of_element
    raise TimeoutException(
 Device 2: Text by xpath:`//android.view.ViewGroup[@content-desc='chat-item']//android.widget.TextView[contains(@text,'https://status.app/c/')]` is not found on the screen after wait_for_visibility_of_element 
```

<b>[[Can't navigate to a community channel before joining the community, https://github.com/status-im/status-mobile/issues/21776]]</b><br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/7f398016b29749b68d18fea6575c3fc3?auth=027c012c7e830c4946eb916a98a1f171#322">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/662c4df8b87e4388adb93a5bd9117ece?auth=be8d26dc7a2852d597406f6c012843e6#328">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3749322">test_community_join_when_node_owner_offline</a>, id: 703629 </b></td></tr><tr><td><p><blockquote><div>Device 2: Tap on found: Button </div><div>Device 2: Looking for community: 'open community' </div></blockquote></p>

```
critical/chats/test_public_chat_browsing.py:1212: in test_community_join_when_node_owner_offline
    self.errors.verify_no_errors()
base_test_case.py:192: in verify_no_errors
    pytest.fail('\n '.join([self.errors.pop(0) for _ in range(len(self.errors))]))
 Joined status is not displayed
E    open community is not listed inside Joined communities tab 
```

<b>[[Can't navigate to a community channel before joining the community, https://github.com/status-im/status-mobile/issues/21776]]</b><br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/7f398016b29749b68d18fea6575c3fc3?auth=027c012c7e830c4946eb916a98a1f171#374">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/662c4df8b87e4388adb93a5bd9117ece?auth=be8d26dc7a2852d597406f6c012843e6#356">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table></details><h3>Passed tests (27)</h3><details><summary>Click to expand</summary><br/><br/><h4>Class TestWalletOneDevice:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3749327">test_wallet_add_remove_regular_account</a>, id: 727231 </b></td></tr><tr><td></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3749332">test_wallet_balance_mainnet</a>, id: 740490 </b></td></tr><tr><td></td></tr></tbody></table><h4>Class TestFallbackMultipleDevice:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3749329">test_fallback_sync_with_error</a>, id: 740220 </b></td></tr><tr><td></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3749331">test_fallback_validate_seed_phrase</a>, id: 740222 </b></td></tr><tr><td></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3749333">test_fallback_add_key_pair</a>, id: 741054 </b></td></tr><tr><td></td></tr></tbody></table><h4>Class TestGroupChatMultipleDeviceMergedNewUI:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3749280">test_group_chat_pin_messages</a>, id: 702732 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/19ef8f2b0a3e4705b9e25fa30a1005b4?auth=f72fdd9ace474448b32e5398f62108fc#288">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/04ef7f820f79423690880f4ee633348c?auth=145e05055aeed9e4e28d8b31d9c21d63#222">Steps, video, logs</a></li></ul></p><p>Device 3:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/cba08f4004dc4692a7c95941876e07cb?auth=d6175b9d2817c717890fcb34f08c7444#236">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3749319">test_group_chat_mute_chat</a>, id: 703495 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/19ef8f2b0a3e4705b9e25fa30a1005b4?auth=f72fdd9ace474448b32e5398f62108fc#416">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/04ef7f820f79423690880f4ee633348c?auth=145e05055aeed9e4e28d8b31d9c21d63#267">Steps, video, logs</a></li></ul></p><p>Device 3:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/cba08f4004dc4692a7c95941876e07cb?auth=d6175b9d2817c717890fcb34f08c7444#251">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3749316">test_group_chat_send_image_save_and_share</a>, id: 703297 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/19ef8f2b0a3e4705b9e25fa30a1005b4?auth=f72fdd9ace474448b32e5398f62108fc#216">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/04ef7f820f79423690880f4ee633348c?auth=145e05055aeed9e4e28d8b31d9c21d63#157">Steps, video, logs</a></li></ul></p><p>Device 3:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/cba08f4004dc4692a7c95941876e07cb?auth=d6175b9d2817c717890fcb34f08c7444#137">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>4. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3749290">test_group_chat_join_send_text_messages_push</a>, id: 702807 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/19ef8f2b0a3e4705b9e25fa30a1005b4?auth=f72fdd9ace474448b32e5398f62108fc#170">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/04ef7f820f79423690880f4ee633348c?auth=145e05055aeed9e4e28d8b31d9c21d63#89">Steps, video, logs</a></li></ul></p><p>Device 3:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/cba08f4004dc4692a7c95941876e07cb?auth=d6175b9d2817c717890fcb34f08c7444#94">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>5. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3749291">test_group_chat_offline_pn</a>, id: 702808 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/19ef8f2b0a3e4705b9e25fa30a1005b4?auth=f72fdd9ace474448b32e5398f62108fc#254">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/04ef7f820f79423690880f4ee633348c?auth=145e05055aeed9e4e28d8b31d9c21d63#194">Steps, video, logs</a></li></ul></p><p>Device 3:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/cba08f4004dc4692a7c95941876e07cb?auth=d6175b9d2817c717890fcb34f08c7444#208">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestDeepLinksOneDevice:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3749323">test_links_open_universal_links_from_chat</a>, id: 704613 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/ea0cb066e24f4644a42c235499d08847?auth=6718b9f310311d911d2595c097b85266#54">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3749284">test_links_deep_links_profile</a>, id: 702775 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/ea0cb066e24f4644a42c235499d08847?auth=6718b9f310311d911d2595c097b85266#118">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3749328">test_deep_links_communities</a>, id: 739307 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/ea0cb066e24f4644a42c235499d08847?auth=6718b9f310311d911d2595c097b85266#189">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestActivityCenterContactRequestMultipleDevicePR:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3749285">test_add_contact_field_validation</a>, id: 702777 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/11f23bb88e124433b7b877fc40798a0a?auth=19fb23696ecde0d8d0735ecc8eb6b992#121">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/fa2fced2668b4c21940dc98f57f3995a?auth=3a9b9e76690e05b654ab261d073aca00#181">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestWalletMultipleDevice:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3749325">test_wallet_send_eth</a>, id: 727229 </b></td></tr><tr><td></td></tr></tbody></table><h4>Class TestActivityMultipleDevicePR:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3749308">test_activity_center_reply_read_unread_delete_filter_swipe</a>, id: 702947 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/100b0626f1bb406e8424bab74b8905d8?auth=737beb988f0a0ef1ed07be4ea88103cd#122">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/67b81b5f452b49c5846b7d2bcbe0eac2?auth=c4b661c8e24f080a66e5f565f99f4e70#101">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestCommunityMultipleDeviceMergedTwo:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3749300">test_community_leave</a>, id: 702845 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/7f398016b29749b68d18fea6575c3fc3?auth=027c012c7e830c4946eb916a98a1f171#318">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/662c4df8b87e4388adb93a5bd9117ece?auth=be8d26dc7a2852d597406f6c012843e6#304">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestCommunityOneDeviceMerged:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3749282">test_community_copy_and_paste_message_in_chat_input</a>, id: 702742 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/769d036bfade40e5b1b382f228663350?auth=8fd275a541599d0f0342d555a3f20fed#75">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3749321">test_community_discovery</a>, id: 703503 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/769d036bfade40e5b1b382f228663350?auth=8fd275a541599d0f0342d555a3f20fed#373">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3749306">test_community_undo_delete_message</a>, id: 702869 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/769d036bfade40e5b1b382f228663350?auth=8fd275a541599d0f0342d555a3f20fed#107">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>4. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3749301">test_community_navigate_to_channel_when_relaunch</a>, id: 702846 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/769d036bfade40e5b1b382f228663350?auth=8fd275a541599d0f0342d555a3f20fed#46">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>5. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3749317">test_community_mute_community_and_channel</a>, id: 703382 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/769d036bfade40e5b1b382f228663350?auth=8fd275a541599d0f0342d555a3f20fed#130">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestActivityMultipleDevicePRTwo:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3749310">test_activity_center_mentions</a>, id: 702957 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/0e8cc71c9c8942b483318b2474a1518b?auth=bb61d70b0641846bc9c1752f661fcd18#122">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/e8780f77cb4a44a9ad8cd2c3f53b6f51?auth=4f80529903cf9280a9731402ea8ed3f6#101">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestOneToOneChatMultipleSharedDevicesNewUi:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3749293">test_1_1_chat_push_emoji</a>, id: 702813 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/e491fe32e7204d4e8b6c53d522986406?auth=e567520243dd6ecc15ac7c5fd493c2d8#308">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/51d94cb742df4405854d41116ea44dca?auth=c930d15411f24e910f99c78a97d264ee#247">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3749318">test_1_1_chat_send_image_save_and_share</a>, id: 703391 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/e491fe32e7204d4e8b6c53d522986406?auth=e567520243dd6ecc15ac7c5fd493c2d8#387">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/51d94cb742df4405854d41116ea44dca?auth=c930d15411f24e910f99c78a97d264ee#341">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestOneToOneChatMultipleSharedDevicesNewUiTwo:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3749288">test_1_1_chat_delete_via_long_press_relogin</a>, id: 702784 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/3b3ec6cf1ff94d7a8e1c7e8106d2b765?auth=b117376a1c76afee3c638413797e88e1#151">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/862b288c21e14fde93858cb9a8c1710a?auth=8b69279b4f21031defba401367a75499#133">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3749287">test_1_1_chat_is_shown_message_sent_delivered_from_offline</a>, id: 702783 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/3b3ec6cf1ff94d7a8e1c7e8106d2b765?auth=b117376a1c76afee3c638413797e88e1#103">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/862b288c21e14fde93858cb9a8c1710a?auth=8b69279b4f21031defba401367a75499#95">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table></details>

--- a/src/legacy/status_im/ui/screens/log_level_settings/views.cljs
+++ b/src/legacy/status_im/ui/screens/log_level_settings/views.cljs
@@ -33,6 +33,9 @@
         name]]]]))
 
 (def log-levels
+  "The status-go log library (zap) doesn't support the trace level, so we remove
+  the trace option from the UI. When trace is enabled an error will happen while
+  trying to login (user will see the error 'wrong password')."
   [{:name  "DISABLED"
     :value ""}
    {:name  "ERROR"
@@ -42,9 +45,7 @@
    {:name  "INFO"
     :value "INFO"}
    {:name  "DEBUG"
-    :value "DEBUG"}
-   {:name  "TRACE"
-    :value "TRACE"}])
+    :value "DEBUG"}])
 
 (views/defview log-level-settings
   []

--- a/status-go-version.json
+++ b/status-go-version.json
@@ -4,6 +4,6 @@
     "owner": "status-im",
     "repo": "status-go",
     "version": "release/7.1.x",
-    "commit-sha1": "d07e61f6157dce56324a1c6814f31515b62eee50",
-    "src-sha256": "13gc362yw8cz35x44gg5j8mjassp3a747v48528d3hv9dzi1vdc6"
+    "commit-sha1": "d3fd630db9223207d36144371bd6b0c85ee24d62",
+    "src-sha256": "00fwhk69i1hkblsykkfqvfid3icx6n77pkmxng5dhq4yc0775l8x"
 }

